story/grammars/fantasy_rules.json

```diff
@@ -34,14 +34,14 @@
     "nice_home" : ["keep", "castle", "manor", "mansion", "abbey", "estate", "tower"],
     "he_she" : ["he", "she"],
     "a_the" : ["a", "the", "your"],
-    "attacked" : ["under attack", "being invaded", "in danger", "being surrounded"],
+    "attacked" : ["under attack", "being invaded", "in danger", "being surrounded", "being sieged"],
     "noble_one" : ["One morning, you are awakened by one of your #subordinate.s#. #he_she.capitalize# tells you that your #nice_home# is #attacked#. You look out #a_the# window and see"],
 
     "element" : ["fire", "water", "earth", "air"],
     "more_element" : ["the underworld", "the sea", "the heavens"],
     "celestial" : ["the sun", "the moon", "the stars"],
-    "domain" : ["#element#", "#more_element#", "#celestial#", "harvest", "fertility", "war", "good fortune", "thresholds", "love"],
-    "god" : ["god", "deity"],
+    "domain" : ["#element#", "#more_element#", "#celestial#", "harvest", "fertility", "war", "good fortune", "thresholds", "love", "wisdom", "#profession.s#"],
+    "god" : ["god", "deity", "godess"],
```

Comment:
Minor spelling error: `godess` should be `goddess`

Reply:
Thanks, corrected.